src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
Thanks, that makes sense. So you're not using cookies right? If you switch them off doesn't that get rid of the error:
```php
ini_set("session.use_cookies", 0);
session_cache_limiter("");
```